opendevin/core/schema/config.py

```diff
@@ -32,6 +32,8 @@ class ConfigType(str, Enum):
     AGENT_MEMORY_MAX_THREADS = 'AGENT_MEMORY_MAX_THREADS'
     AGENT_MEMORY_ENABLED = 'AGENT_MEMORY_ENABLED'
     MAX_ITERATIONS = 'MAX_ITERATIONS'
+    MAX_ITERATIONS_PER_TASK = 'MAX_ITERATIONS_PER_TASK'
+    MAX_CHARS = 'MAX_CHARS'
```

Comment:
I think we removed MAX_CHARS the other day

Reply:
yes upstream didnt have it. I just checked. Took a note of this too!